lgc/elfLinker/GlueShader.cpp

```diff
@@ -451,3 +448,14 @@ Function *ColorExportShader::createColorExportFunc() {
   builder.CreateRetVoid();
   return func;
 }
+
+void ColorExportShader::updatePalMetadata(PalMetadata *palMetadata) {
```

Comment:
Please add the function header comments.

Reply:
Done.